malariagen_data/anoph/dipclust.py

```diff
@@ -631,7 +631,7 @@ def plot_diplotype_clustering_advanced(
                 subplot_heights.append(cnv_row_height * n_cnv_genes)
 
         if snp_transcript:
-            snp_trace, n_snps = self._dipclust_snp_trace(
+            snp_trace, n_snps_df = self._dipclust_snp_trace(
```

Comment:
```suggestion
            snp_trace, n_snps_transcript = self._dipclust_snp_trace(
```

Nit, suggest renaming for even less ambiguity :)